mkosi/__init__.py

```diff
@@ -1896,6 +1931,7 @@ def build_image(args: MkosiArgs, config: MkosiConfig, name: str, uid: int, gid:
 
             install_build_dest(state)
             install_extra_trees(state)
+            configure_os_release(state)
```

Comment:
Can we do this a bit later actually together with all the other `configure()` functions?

Reply:
I went for the `configure_` name because I couldn't come up with anything, but I wanted to allow the postinst script to be able to see the full os release file already, but I'm fine with having it afterwards, since the environment variables will expose the data, too.